models/user.js

```diff
@@ -0,0 +1,88 @@
+const bcrypt = require('bcrypt');
+//todo: import bcrypt to hash the words before create and before update hook for passwords
+
+//refer to bcrypt documentation and class materials
+
+const { Model, DataTypes } = require('sequelize');
+
+const sequelize = require('../config/connection.js');
+
+class User extends Model {}
```

Comment:
Please add a method definition in here to check a password against the stored password using bcrypt. 

Make sure it is an async function and return await a result from the bcrypt async compare function: 
https://www.npmjs.com/package/bcrypt

Reply:
it is already in the hook, spoke to Chee about it just got to add a validator